wp-content/plugins/core/src/Theme/Image_Sizes.php

```diff
@@ -5,10 +5,18 @@
 
 
 class Image_Sizes {
-	const CORE_FULL      = 'core-full';
-	const CORE_MOBILE    = 'core-mobile';
-	const SOCIAL_SHARE   = 'social-share';
-	const COMPONENT_CARD = 'component-card';
+	const CORE_FULL             = 'core-full';
+	const CORE_MOBILE           = 'core-mobile';
+	const SOCIAL_SHARE          = 'social-share';
+	const COMPONENT_CARD        = 'component-card';
+	const SQUARE_LARGE          = 'square-large';
+	const SQUARE_MEDIUM         = 'square-med';
+	const SQUARE_SMALL          = 'square-sm';
+	const SQUARE_PLACEHOLDER    = 'square-placeholder';
+	const CORE_16X9_LARGE       = '16x9-lg';
+	const CORE_16X9_MEDIUM      = '16x9-med';
+	const CORE_16X9_SMALL       = '16x9-sm';
+	const CORE_16X9_PLACEHOLDER = '16x9-placeholder';
```

Comment:
How are we using this size?

Reply:
I'm using this as a custom blurred shim for lazyloading here: https://github.com/moderntribe/square-one/blob/232faea4214dfa28ff17bd9c18523a67ba29725a/wp-content/plugins/core/src/Templates/Content/Panels/Hero.php#L83

Reply:
Ah, ok, we want to reassess our lazyloading handling in general (with native browser support coming), including for blurry vs. what we do now, so that's cool, a step in that direction :)